woocommerce/rest-api/class-sv-wc-plugin-rest-api.php

```diff
@@ -134,12 +154,87 @@ protected function get_system_status_data() {
 	 */
 	public function register_routes() {
 
-		// stub
+		foreach ( $this->endpoints as $endpoint => $versions ) {
+
+			foreach ( $versions as $version ) {
+
+				if ( 'debug' === $endpoint && ( $debug = $this->get_debug_controller_instance( $version ) ) ) {
+
+					$debug->register_routes();
+
+				} elseif ( 'log' === $endpoint && ( $log = $this->get_log_controller_instance( $version ) ) ) {
+
+					$log->register_routes();
+				}
+			}
+		}
+	}
+
+
+	/**
+	 * Gets the debug controller endpoint handler instance.
+	 *
+	 * @since 5.5.0-dev
+	 *
+	 * @param string $api_version API version for the controller to be returned
+	 * @return null|Debug_Controller
+	 */
+	public function get_debug_controller_instance( $api_version = 'v3' ) {
```

Comment:
one thing to be mindful of is that we need to give plugins the ability to override the controller class easily, in case they need to use their own child class instead.

Perhaps we could use an additional method to get the controller class name, which would be overridable without having to duplicate the logic inside this method?